src/stability_sdk/client.py

```diff
@@ -318,8 +387,61 @@ def upscale(
 
         Variables ignored in ESRGAN engines: prompt, steps, cfg_scale, seed
 
-        :return: Tuple of (prompts, image_parameters)
+        :return: Generator of Answer objects.
+        """
+        rq = self._make_upscale_request(
+                init_image=init_image,
+                height=height,
+                width=width,
+                prompt=prompt,
+                steps=steps,
+                cfg_scale=cfg_scale,
+                seed=seed
+        )
+
+        yield from self.run_request(rq)
+    
+    def _make_upscale_request(
+        self,
+        init_image: Image.Image = None,
+        height: int = None,
+        width: int = None,
+        prompt: Union[str, generation.Prompt] = None,
+        steps: Optional[int] = 20,
+        cfg_scale: float = 7.0,
+        seed: int = 0
+    ) -> generation.Request:
         """
+        Create an upscale request for a chain of engines.
```

Comment:
Having matching docstrings on upscale and _make_upscale_request seems redundant and need to be kept in sync. Maybe skip the docstring on the internal method or have it refer the reader to the upscale method?